playbooks/roles/nginx/templates/edx/app/nginx/sites-available/kibana.j2

```diff
@@ -27,9 +27,6 @@ server {
 
   root {{ kibana_app_dir }}/htdocs;
 
-  access_log {{ nginx_log_dir }}/kibana.access.log;
```

Comment:
@rachelg4 Do we need to delete this here? Could we leave it in place and then _only_ override it where we need it (in `location /heartbeat`)?

Not only would that be less code (just adding a new stanza to `/heartbeat` instead of adding 3 stanzas and removing another), but it's less of a maintenance burden.

As is, this approach would require us to copy/paste these two lines into _every_ new stanza we add (someone will inevitably forget), and we risk inadvertently sending the logs to an incorrect location (probably the nginx default location).